src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2767,7 +2767,7 @@ static bool contextElementPreconditionsCheck
       {
         ContextAttribute* ca = new ContextAttribute(ceP->contextAttributeVector.get(ix));
         buildGeneralErrorResponse(ceP, ca, responseP, SccInvalidModification,
-                                  "duplicated attribute name and id [" + name + "," + id + "]");
+                                  "duplicated attribute /" + name + "/");
         LM_W(("Bad Input (duplicated attribute name: name=<%s> id=<%s>)", name.c_str(), id.c_str()));
         return false; // Error already in responseP
```

Comment:
I just realized a thing ...

    ContextAttribute* ca = new ContextAttribute( ... );
    ...
    return false;

This must be a leak, right?
An 'old' one but not SO old ...

Easy fix;

    ContextAttribute* ca = ceP->contextAttributeVector.get(ix);